how2meet/db/database.py

```diff
@@ -11,16 +11,18 @@
 from sqlalchemy.orm import Session, declarative_base, sessionmaker
 
 # TODO: connect to postres instance from env
-SQLALCHEMY_DATABASE_URL = os.getenv("DB_CONN", "sqlite:///./data.db")
+SQLALCHEMY_DATABASE_URL = os.getenv("DB_CONN", "")
 
 if SQLALCHEMY_DATABASE_URL.startswith("postgres://"):
     # Replace postgres:// with postgresql:// due to SQLAlchemy issue with Render's autogenerated db url
     # Ref: https://community.render.com/t/sqlalchemy-exc-nosuchmoduleerror-cant-load-plugin-sqlalchemy-dialects-postgres/11789
     # Ref: https://github.com/sqlalchemy/sqlalchemy/issues/6083#issuecomment-821914503
     SQLALCHEMY_DATABASE_URL = SQLALCHEMY_DATABASE_URL.replace("postgres://", "postgresql://")
     engine = create_engine(SQLALCHEMY_DATABASE_URL, echo=True)  # Postgres db
-
+elif SQLALCHEMY_DATABASE_URL.startswith("postgresql://"):
```

Comment:
I get that this will work for a local Postgres db, but I'm concerned it's going to cause an issue in the future given that `postgres` and `postgresql` are so close. Maybe we ought to set an envvar to distinguish between a local db and a hosted db?

Reply:
There isn't actually anything different being done though. All this code is doing is skipping your workaround for the erroneous render connection string. Local and hosted have the same behavior here.